CDP4WebServices.API/Services/Authorization/ObfuscationService.cs

```diff
@@ -80,6 +88,8 @@ public void ObfuscateResponse(List<Thing> resourceResponse, Credentials credenti
             var citations = resourceResponse.OfType<Citation>().ToList();
             var aliases = resourceResponse.OfType<Alias>().ToList();
             var hyperlinks = resourceResponse.OfType<HyperLink>().ToList();
+            var modellogitems = resourceResponse.OfType<ModelLogEntry>().ToList();
+            var logentries = resourceResponse.OfType<LogEntryChangelogItem>().ToList();
```

Comment:
can you rename this variable to `logEntryChangelogItems` to make it more explicit please?